docs/API Reference Documentation/Policy Table Update/index.md

```diff
@@ -12,9 +12,9 @@ Policy Table Update Sequence Diagram
 3. The RPC's data is, optionally, encrypted using an asynchronous key that only the Policy Server can decrypt.  The URL and headers are not encrypted since they are required by the mobile library to forward the request to the Policy Server.
 4. The RPC is then sent to the mobile library.
 5. The mobile library will ignore the request body containing the Policy Table snapshot, because it is marked as proprietary, and will forward the request to the URL included in the OnSystemRequest RPC.  If the request fails to send then the mobile library will attempt to retry using the configuration specified in the [module config](../policy-table/module-config).
-6. When the server receives the Policy Table update request it will first lookup the module in the server's database using a unique identifier.  If the module is not found an error will be retured in the server's response.
+6. When the server receives the Policy Table update request it will first lookup the module in the server's database using a unique identifier.  If the module is not found an error will be returned in the server's response.
```

Comment:
```suggestion
6. When the server receives the Policy Table update request it will first look up the module in the server's database using a unique identifier.  If the module is not found an error will be returned in the server's response.
```